### ISSUE TITLE
Regression: Spinner no longer toggleable with CSS

### ISSUE THREAD

**Issue by pouretrebelle** [primer/view_components#919]
https://github.com/primer/view_components/pull/845 wrapped the spinner svg in a `span` to add a11y features, but this has caused an issue in .com where a sibling selector was used to toggle the spinner on/off [[see this Slack thread](https://github.slack.com/archives/CMZ4DC9BL/p1638179917092500)].

Passing a class into Spinner renders the class on the child svg element instead of the wrapping span element, so although toggling it with `display: none/block` would show/hide the spinner, the a11y advantages are lost because it's not hiding/showing the loading message itself - in fact it's worse than when there was no loading label because it'll announce loading as soon as the element is rendered.

Ideally the whole element would be added and removed from the DOM when needed, but since a spinner is almost always a side effect of something else it's probably much more common to toggle them using CSS, as in the issue linked above.
